docs/README.cursor.md

```
@@ -0,0 +1,220 @@
+
+```bash
+ln -s ../superpowers/skills .cursor/skills/superpowers
+```
+
+##### 4. Copy Superpowers Rule
+
+```bash
+mkdir -p .cursor/rules
+cp .cursor/superpowers/.cursor/rules/superpowers.mdc .cursor/rules/superpowers.mdc
+```
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)